.github/workflows/bench_summary_comment.yml

```diff
@@ -0,0 +1,35 @@
+name: Bench output summary
+
+on:
+  issue_comment:
+    types: created
+
+jobs:
+  Produce_bench_summary:
+    name: Post summary of benchmarking results
+    if: github.event.issue.pull_request && (startsWith(github.event.comment.body, 'Here are the [benchmark results]'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: install elan
+        run: |
+          set -o pipefail
+          curl -sSfL https://github.com/leanprover/elan/releases/download/v3.1.1/elan-x86_64-unknown-linux-gnu.tar.gz | tar xz
+          ./elan-init -y --default-toolchain none
+          echo "$HOME/.elan/bin" >> "${GITHUB_PATH}"
+
+      - uses: actions/checkout@v4
+        with:
+          ref: master
+          sparse-checkout: |
+            scripts/bench_summary.lean
+
+      - name: Summarize bench output
+        run: |
+          {
```

Comment:
One of my first thoughts was: "surely this can be rewritten as a `script`, callable via `lake exe`, right"? Right now, the answer is no (because the script uses `IO.Log`, which requires a stronger monad than just `IO`) --- but this is just for debugging, right? (You *could* sneakily use dbg_trace, I presume...?)

Put differently: the current set-up is clever, but I wonder if something more standard is possible.

Reply:
I'll happily push a commit rewriting this as a script, if you think that is good.

Reply:
A script would be nice, but maybe I would wait to see if there is interest in actually running this script outside of CI.  Btw, using `IO.print`/`IO.println` may be better than `dbg_trace`, assuming that they are in the right monad.

Reply:
Good point. I was thinking about this mainly from a maintainance point of view: adding a script is well-understood - but this can easily be a follow-up.

Sure, `IO.println` would work (you just need an output file...).